### DISCUSSION TITLE
Compilation Error with VS2022 community

### DISCUSSION THREAD

**Discussion by pb07** [sumatrapdfreader/sumatrapdf#3710] [Q&A] [answered]
![CopieEcran-20230923-105523837-4](https://github.com/sumatrapdfreader/sumatrapdf/assets/350599/39a40fe7-1932-401d-86aa-e6c9883a0e87)

Is any code or library missing from the project  ( version 3.5 ) ?

**Comment by kjk** [comment, marked as answer]
no
